app/js/lib/foxbox/foxbox.js

```diff
@@ -466,4 +468,29 @@ export default class Foxbox extends Service {
         });
       });
   }
+
+  /**
+   * Returns value type string for the specified operation kind.
+   * @param {string|Object} operationKind Kind of the operation, string for the
+   * well known type and object for the Extension channel kind.
+   * @returns {string}
```

Comment:
I usually use `@return` instead of `@returns` which is supported by Closure Compiler.